velox/connectors/hive/storage_adapters/abfs/AbfsReadFile.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <azure/storage/blobs/blob_client.hpp>
```

Comment:
Move this header to the CPP file and forward declare  `BlobClient`
```
namespace Azure::Storage::Blobs {
  class BlobClient;
}
```
We should try to minimize including external headers or else we will leak them into the client. That will add a dependency for the client on the external package.

Reply:
@majetideepak I didn't find a easy way to move header to CPP without changing the AbfsReadFile constructor definitation.
I eventually follow how S3 read/writefile implement and move all to AbfsFileSystem cpp. Hope it also a acceptable solution.